Dockerfile

```diff
@@ -45,8 +45,12 @@ COPY . /src/
 RUN rm -rf /src/build
 RUN mkdir -p /src/build
 
-WORKDIR /src/build
+# sonarqube stuff
+WORKDIR /src
```

Comment:
Siehe GameLogic: ist es nicht möglich das erst nachher alles in einem Skript zu machen, so dass Docker auch zum normalen Deploy komplett ohne SQ genutzt werden kann